apps/06SpaceTravelToMars.js

```diff
@@ -9,7 +9,37 @@ var jobTypes = {
   programmer: 'Any Ship!'
 };
 
-// Your code here
+function CrewMember(name, job, specialSkill) {
+  this.name = name;
+  this.job = job;
+  this.specialSkill = specialSkill;
+  //this.ship = null;
+  this.enterShip = function(ship){
+    this.ship = ship;
+    ship.crew.push(this)
+  }
+}
+
+var rick = new CrewMember('Rick Martinez', 'pilot', 'chemistry');
```

Comment:
Well originally my thinking was, whoosh there are a bunch of crewMembers, I'm going to need to create variables for ALL of them!! So I wrote statements for them using celebrities: Matthew McCaughey, Oprah, Obama, and Kevin James... It was going to be 1 mad crazy interesting trip to Mars! lol